src/converters/lintConfigs/rules/convertRules.test.ts

```diff
@@ -201,7 +278,8 @@ describe("convertRules", () => {
         );
 
         // Assert
-        expect(converted).toEqual(
+        expect([
+            // don't care if arguments merger ran or not
```

Comment:
Nit: I think we can do away with these two extra comment lines; they're implied by the tests' behavior.

```suggestion
```